mixins.scss

```diff
@@ -884,11 +939,11 @@
 			$unit: unit($value);
 
 			@if $unit == 'px' {
-				$px-values: append($px-values, #{strip-unit($value)}px );
+				$px-values: append($px-values, #{strip-unit($value)} * 1px );
 				$rem-values: append($rem-values, (strip-unit($value) / strip-unit($main-font-size) * 1rem));
 			} @else {
 				$px-values: append($px-values, ($value * $main-font-size) );
-				$rem-values: append($rem-values, #{$value}rem);
+				$rem-values: append($rem-values, #{$value} * 1rem);
```

Comment:
I would change that in another PR, because i will concentrate here only on the documentation